### DISCUSSION TITLE
search for additional information for products

### DISCUSSION THREAD

**Discussion by ssalipro** [pimcore/pimcore#7815] [Q&A / Support]
i would like to modify the frontend search for products to include searching for text in the additional information for products.
for example in the pimcore demo i would like to search for additional information about cars like search for cars produced in a given year like 1965 and should get those results in the frontend.

**Comment by brusch** [comment]
Please have a look at the [Index](https://pimcore.com/docs/pimcore/current/Development_Documentation/E-Commerce_Framework/Index_Service/index.html) & [Filter](https://pimcore.com/docs/pimcore/current/Development_Documentation/E-Commerce_Framework/Filter_Service/index.html) Service of the Ecommerce Framework.
